2016/06/10 01:22:58 [error] [exception.CException] exception 'CException' with message 'Property "LocalController.widget" is not defined.' in D:\OpenServer\domains\blog\yii\framework\base\CComponent.php:130
Stack trace:
#0 D:\OpenServer\domains\blog\protected\views\local\index.php(22): CComponent->__get('widget')
#1 D:\OpenServer\domains\blog\yii\framework\web\CBaseController.php(126): require('D:\\OpenServer\\d...')
#2 D:\OpenServer\domains\blog\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\\OpenServer\\d...', Array, true)
#3 D:\OpenServer\domains\blog\yii\framework\web\CController.php(869): CBaseController->renderFile('D:\\OpenServer\\d...', Array, true)
#4 D:\OpenServer\domains\blog\yii\framework\web\CController.php(782): CController->renderPartial('index', Array, true)
#5 D:\OpenServer\domains\blog\protected\controllers\LocalController.php(128): CController->render('index', Array)
#6 D:\OpenServer\domains\blog\yii\framework\web\actions\CInlineAction.php(49): LocalController->actionIndex()
#7 D:\OpenServer\domains\blog\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 D:\OpenServer\domains\blog\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#9 D:\OpenServer\domains\blog\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#10 D:\OpenServer\domains\blog\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#11 D:\OpenServer\domains\blog\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#12 D:\OpenServer\domains\blog\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#13 D:\OpenServer\domains\blog\yii\framework\web\CController.php(291): CFilterChain->run()
#14 D:\OpenServer\domains\blog\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 D:\OpenServer\domains\blog\yii\framework\web\CWebApplication.php(282): CController->run('index')
#16 D:\OpenServer\domains\blog\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('local/index')
#17 D:\OpenServer\domains\blog\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#18 D:\OpenServer\domains\blog\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/index.php?r=local/index
---
2016/06/10 01:53:33 [error] [php] implode(): Invalid arguments passed (D:\OpenServer\domains\blog\protected\views\site\contact.php:6)
Stack trace:
#0 D:\OpenServer\domains\blog\yii\framework\web\CBaseController.php(95): SiteController->renderInternal()
#1 D:\OpenServer\domains\blog\yii\framework\web\CController.php(869): SiteController->renderFile()
#2 D:\OpenServer\domains\blog\yii\framework\web\CController.php(782): SiteController->renderPartial()
#3 D:\OpenServer\domains\blog\protected\controllers\SiteController.php(72): SiteController->render()
#4 D:\OpenServer\domains\blog\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionContact()
#5 D:\OpenServer\domains\blog\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#6 D:\OpenServer\domains\blog\yii\framework\web\CController.php(286): SiteController->runAction()
#7 D:\OpenServer\domains\blog\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#8 D:\OpenServer\domains\blog\yii\framework\web\CWebApplication.php(282): SiteController->run()
#9 D:\OpenServer\domains\blog\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#10 D:\OpenServer\domains\blog\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#11 D:\OpenServer\domains\blog\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=site/contact
in D:\OpenServer\domains\blog\protected\views\site\contact.php (6)
in D:\OpenServer\domains\blog\protected\controllers\SiteController.php (72)
in D:\OpenServer\domains\blog\index.php (13)
2016/06/10 01:54:09 [error] [php] implode(): Invalid arguments passed (D:\OpenServer\domains\blog\protected\views\site\contact.php:6)
Stack trace:
#0 D:\OpenServer\domains\blog\yii\framework\web\CBaseController.php(95): SiteController->renderInternal()
#1 D:\OpenServer\domains\blog\yii\framework\web\CController.php(869): SiteController->renderFile()
#2 D:\OpenServer\domains\blog\yii\framework\web\CController.php(782): SiteController->renderPartial()
#3 D:\OpenServer\domains\blog\protected\controllers\SiteController.php(72): SiteController->render()
#4 D:\OpenServer\domains\blog\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionContact()
#5 D:\OpenServer\domains\blog\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#6 D:\OpenServer\domains\blog\yii\framework\web\CController.php(286): SiteController->runAction()
#7 D:\OpenServer\domains\blog\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#8 D:\OpenServer\domains\blog\yii\framework\web\CWebApplication.php(282): SiteController->run()
#9 D:\OpenServer\domains\blog\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#10 D:\OpenServer\domains\blog\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#11 D:\OpenServer\domains\blog\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=site/contact
in D:\OpenServer\domains\blog\protected\views\site\contact.php (6)
in D:\OpenServer\domains\blog\protected\controllers\SiteController.php (72)
in D:\OpenServer\domains\blog\index.php (13)
2016/06/10 01:54:11 [error] [php] implode(): Invalid arguments passed (D:\OpenServer\domains\blog\protected\views\site\contact.php:6)
Stack trace:
#0 D:\OpenServer\domains\blog\yii\framework\web\CBaseController.php(95): SiteController->renderInternal()
#1 D:\OpenServer\domains\blog\yii\framework\web\CController.php(869): SiteController->renderFile()
#2 D:\OpenServer\domains\blog\yii\framework\web\CController.php(782): SiteController->renderPartial()
#3 D:\OpenServer\domains\blog\protected\controllers\SiteController.php(72): SiteController->render()
#4 D:\OpenServer\domains\blog\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionContact()
#5 D:\OpenServer\domains\blog\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#6 D:\OpenServer\domains\blog\yii\framework\web\CController.php(286): SiteController->runAction()
#7 D:\OpenServer\domains\blog\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#8 D:\OpenServer\domains\blog\yii\framework\web\CWebApplication.php(282): SiteController->run()
#9 D:\OpenServer\domains\blog\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#10 D:\OpenServer\domains\blog\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#11 D:\OpenServer\domains\blog\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=site/contact
in D:\OpenServer\domains\blog\protected\views\site\contact.php (6)
in D:\OpenServer\domains\blog\protected\controllers\SiteController.php (72)
in D:\OpenServer\domains\blog\index.php (13)
2016/06/10 01:55:03 [error] [php] implode(): Invalid arguments passed (D:\OpenServer\domains\blog\protected\views\site\contact.php:6)
Stack trace:
#0 D:\OpenServer\domains\blog\yii\framework\web\CBaseController.php(95): SiteController->renderInternal()
#1 D:\OpenServer\domains\blog\yii\framework\web\CController.php(869): SiteController->renderFile()
#2 D:\OpenServer\domains\blog\yii\framework\web\CController.php(782): SiteController->renderPartial()
#3 D:\OpenServer\domains\blog\protected\controllers\SiteController.php(72): SiteController->render()
#4 D:\OpenServer\domains\blog\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionContact()
#5 D:\OpenServer\domains\blog\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#6 D:\OpenServer\domains\blog\yii\framework\web\CController.php(286): SiteController->runAction()
#7 D:\OpenServer\domains\blog\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#8 D:\OpenServer\domains\blog\yii\framework\web\CWebApplication.php(282): SiteController->run()
#9 D:\OpenServer\domains\blog\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#10 D:\OpenServer\domains\blog\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#11 D:\OpenServer\domains\blog\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=site/contact
in D:\OpenServer\domains\blog\protected\views\site\contact.php (6)
in D:\OpenServer\domains\blog\protected\controllers\SiteController.php (72)
in D:\OpenServer\domains\blog\index.php (13)
2016/06/10 02:04:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "post".' in D:\OpenServer\domains\blog\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\OpenServer\domains\blog\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post')
#1 D:\OpenServer\domains\blog\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 D:\OpenServer\domains\blog\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=post
---
2016/06/10 02:13:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "_form".' in D:\OpenServer\domains\blog\yii\framework\web\CController.php:483
Stack trace:
#0 D:\OpenServer\domains\blog\yii\framework\web\CController.php(270): CController->missingAction('_form')
#1 D:\OpenServer\domains\blog\yii\framework\web\CWebApplication.php(282): CController->run('_form')
#2 D:\OpenServer\domains\blog\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('local/_form')
#3 D:\OpenServer\domains\blog\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 D:\OpenServer\domains\blog\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=local/_form
---
2016/06/10 02:29:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "about" was not found.' in D:\OpenServer\domains\blog\yii\framework\web\actions\CViewAction.php:112
Stack trace:
#0 D:\OpenServer\domains\blog\yii\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('about')
#1 D:\OpenServer\domains\blog\yii\framework\web\actions\CAction.php(76): CViewAction->run()
#2 D:\OpenServer\domains\blog\yii\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 D:\OpenServer\domains\blog\yii\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 D:\OpenServer\domains\blog\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 D:\OpenServer\domains\blog\yii\framework\web\CWebApplication.php(282): CController->run('page')
#6 D:\OpenServer\domains\blog\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 D:\OpenServer\domains\blog\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#8 D:\OpenServer\domains\blog\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/index.php?r=site/page&view=about
HTTP_REFERER=http://blog/index.php?r=site/index
---
2016/06/10 02:40:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "index.php?r=local/create".' in D:\OpenServer\domains\blog\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\OpenServer\domains\blog\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('index.php?r=loc...')
#1 D:\OpenServer\domains\blog\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 D:\OpenServer\domains\blog\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=index.php?r=local/create
HTTP_REFERER=http://blog/index.php?r=local/create
---
2016/06/10 02:41:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "?r=local/create".' in D:\OpenServer\domains\blog\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\OpenServer\domains\blog\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('?r=local/create')
#1 D:\OpenServer\domains\blog\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 D:\OpenServer\domains\blog\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=?r=local/create
HTTP_REFERER=http://blog/index.php?r=local/create
---
2016/06/10 02:41:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "?r=local/create".' in D:\OpenServer\domains\blog\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\OpenServer\domains\blog\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('?r=local/create')
#1 D:\OpenServer\domains\blog\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 D:\OpenServer\domains\blog\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=?r=local/create
HTTP_REFERER=http://blog/index.php?r=local/create
---
